pythonTools/mbuildlib/projectwriter.py

```diff
@@ -4,6 +4,9 @@
 import os
 import posixpath
 
+if platform.system() == 'Windows':
+    import winreg ## can now safely import
```

Comment:
This needs to be verified in testing, but it may be that here pyreq needs to be also called. It behavior like R's require(). `pyreq.require("winreg")` before `import winreg`

Reply:
I believe since I moved `from mbuildlib.projectwriter import [...]` to be after the `pyreq.require("winreg")` in mbuild.py itself, it should be fine but definitely will need testing.